third_party/fmt-6.1.2/include/fmt/core.h

```diff
@@ -233,7 +233,7 @@ FMT_API void assert_fail(const char* file, int line, const char* message);
 #    define FMT_ASSERT(condition, message) \
       ((condition)                         \
            ? void()                        \
-           : fmt::internal::assert_fail(__FILE__, __LINE__, (message)))
+           : ::fmt::internal::assert_fail(__FILE__, __LINE__, (message)))
```

Comment:
@lefticus Thanks for the proper fix. Confirmed this branch now builds locally for debug config on windows.